internal/store/projects.go

```diff
@@ -106,16 +106,20 @@ func (s *Store) Project(id string) (interface{}, error) {
 }
 
 // CreateProject encapsulates the logic to manage different cloud providers.
-func (s *Store) CreateProject(name, orgID, regionUsageRestrictions string, opts *atlas.CreateProjectOptions) (interface{}, error) {
+func (s *Store) CreateProject(name, orgID, regionUsageRestrictions string, defaultAlertSettings bool, opts *atlas.CreateProjectOptions) (interface{}, error) {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
-		project := &atlas.Project{Name: name, OrgID: orgID, RegionUsageRestrictions: regionUsageRestrictions}
+		project := &atlas.Project{Name: name, OrgID: orgID, RegionUsageRestrictions: regionUsageRestrictions, WithDefaultAlertsSettings: &defaultAlertSettings}
 		result, _, err := s.client.(*atlas.Client).Projects.Create(s.ctx, project, opts)
 		return result, err
-	case config.CloudManagerService, config.OpsManagerService:
+	case config.OpsManagerService:
 		project := &opsmngr.Project{Name: name, OrgID: orgID}
 		result, _, err := s.client.(*opsmngr.Client).Projects.Create(s.ctx, project, opts)
 		return result, err
```

Comment:
If we don’t separate OM from CM we get this error 

```
./bin/mongocli iam project create test1  -P ops-2
Error: POST http://ec2-34-253-240-91.eu-west-1.compute.amazonaws.com:9080/api/public/v1.0/groups: 400 (request "INVALID_ATTRIBUTE") Invalid attribute withDefaultAlertsSettings specified.
```

Reply:
if you go this way we're going to need to update this when Ops manager 6 comes out, you can keep them together but set `defaultAlertSettings` to eb a pointer in this method and set it to nil if the version of ops manager is older than 6

Reply:
sorry, I have missed this comment. Working on it

Reply:
updated